### PR TITLE
duplicates plot

### PR DESCRIPTION
**Pull request by jduerholt**
This PR adds a plot that can be used to show duplicates in the data as below:

image

### CODE REVIEW

**Comment by bertiqwerty** [comment]
Thanks, Johannes. Can you explain the use case?

**Comment by jduerholt** [comment]
It is one of my typical methods that I use when working with real world datasets and preparing them for BO, I search for the duplicates in the data and plot them like this to see if there are obvious outliers in the duplicates ;)

File: bofire/plot/duplicates.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+def plot_duplicates_plotly(
+    experiments: pd.DataFrame, duplicates: List[str], key: str, scale: float = 0.1
+) -> go.Figure:
+    """
+    Plots duplicates using Plotly.
+
+    Args:
+        experiments: The dataframe containing the experiments data.
+        duplicates: A list of strings representing the labcodes of the duplicates.
```
Comment (bertiqwerty) [inline bofire/plot/duplicates.py:16]: What are labcodes?

Comment (jduerholt) [reply bofire/plot/duplicates.py:16]: Hi @bertiqwerty ,

these is an optional column in the `experiments` data frame which can be a string based unique identifier of one experiment.

Best,

Johannes

**Review by bertiqwerty — APPROVED** [review]

**Pull request merged**